src/microbe_stage/systems/MicrobeEmissionSystem.cs

```diff
@@ -345,11 +356,61 @@ private void HandleSlimeSecretion(in Entity entity, ref MicrobeControl control,
         {
             // Deactivate the jets if we aren't supposed to secrete slime
             foreach (var jet in organelles.SlimeJets)
+            {
                 jet.Active = false;
+            }
         }
 
-        control.QueuedSlimeSecretionTime -= delta;
-        if (control.QueuedSlimeSecretionTime < 0)
-            control.QueuedSlimeSecretionTime = 0;
+        DecreaseTimeCountdownValue(ref control.QueuedSlimeSecretionTime, delta);
+    }
+
+    private void HandleMucusSecretion(in Entity entity, ref MicrobeControl control,
```

Comment:
What's this code about? It seems like you are partially reimplementing the MucocystSystem here, which is not where this code belongs as there's a dedicated system handling mucocyst operations.